Description: initialize variable 'uwsgi_fd'
 gcc (v4.4.5 20100816) emits warning about it when CFLAGS+="-O2"
Author: Leonid Borisenko <leo.borisenko@gmail.com>

Index: uwsgi-0.9.6/http.c
===================================================================
--- uwsgi-0.9.6.orig/http.c	2010-08-29 11:17:38.000000000 +0300
+++ uwsgi-0.9.6/http.c	2010-08-29 11:17:48.000000000 +0300
@@ -182,7 +182,7 @@
 	struct uwsgi_http_req *ur = (struct uwsgi_http_req *) u_h_r ;
 
 	int clientfd = ur->fd;
-	int uwsgi_fd;
+	int uwsgi_fd = 0;
 
 	int need_to_read = 1;
 	int state = uwsgi_http_method;
